package.json

```diff
@@ -45,6 +46,7 @@
     "postinstall": "node ./bin/hello.js",
     "setup": "node ./bin/setup.js",
     "netlify:login": "netlify login",
-    "netlify:deploy": "netlify deploy -d public -p"
+    "netlify:deploy": "netlify deploy -d public -p",
+    "format": "prettier --write 'src/**/*.js' 'bin/*.js'"
```

Comment:
Same here. All upgrades and adding prettier is unrelated to your PR

Reply:
I know - I was confused as to why I needed to add styling stuff (per your comment above), but maybe I am misunderstanding? 

What did you mean when you said:
> I'd like to have the userland version and this one aligned, at least on dependency/codestyle level.

It sounded to me like you wanted me to implement the same code style settings as that userland repo PR, but I'm not really sure what the intent was :sweat_smile: I'll standby for feedback!